test/libsolidity/syntaxTests/types/function_types/function_parameter_return_types_fail.sol

```diff
@@ -0,0 +1,43 @@
+contract Test
+{
+    function uint256_to_uint256(uint256 x) internal pure returns (uint256) { return x; }
+    function uint256_to_string(uint256 x) internal pure returns (string memory) { return x == 0 ? "a" : "b"; }
+    function uint256_to_string_storage(uint256) internal pure returns (string storage);
+    function string_to_uint256(string memory x) internal pure returns (uint256) { return bytes(x).length; }
+    function string_to_string(string memory x) internal pure returns (string memory) { return x; }
+    
+    function uint256_uint256_to_uint256(uint256 x, uint256 y) internal pure returns (uint256) { return x + y; }
+    function uint256_uint256_to_string(uint256 x, uint256 y) internal pure returns (string memory) { return x == y ? "a" : "b"; }
+    function string_uint256_to_string(string memory x, uint256 y) internal pure returns (string memory) { return y == 0 ? "a" : x; }
+    function string_string_to_string(string memory x, string memory y) internal pure returns (string memory) { return bytes(x).length == 0 ? y : x; }
+    function uint256_string_to_string(uint256 x, string memory y) internal pure returns (string memory) { return x == 0 ? "a" : y; }
+    
+    function tests() internal pure
+    {
+    	// Should succeed:
```

Comment:
I think we had the rule to split succeeding and failing tests.

Reply:
I split them but forgot to remove the comment - these actually all fail.